client/src/components/Contact.style.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
+  font-size: 3rem;
+  background-color: var(--highlight);
+  margin-bottom: 3%;
+  font-family: 'Work Sans';
+  font-style: normal;
+  font-weight: 900;
+`;
+export const H2 = styled.h2`
+  font-size: 1rem;
+  color: var(--black);
+  margin-left: 29%;
+  margin-top: 3%;
+  background: var(--highlight);
+  width: 113px;
+  margin-bottom: 6%;
+  font-family: 'Work Sans', sans serif;
+  font-style: normal;
+  font-weight: 900;
+  line-height: 20px;
+  text-align: center;
+  letter-spacing: 0.1em;
+  text-decoration-line: underline;
+`;
+export const Address = styled.div`
```

Comment:
this needs to stilll be changed to:

```export const Address = styled.address` ```

as we are trying not to use divs!

also, here  i might suggest a:

```min-height: 55vh```

Reply:
Did you mean to put  `min-height: 55vh` instead of `height: 220px` ? @teenie-quaggard